drivers/pppos.c

```diff
@@ -451,6 +452,7 @@ static void pppos_mainLoop(void* _state)
 {
 	pppos_priv_t* state = (pppos_priv_t*) _state;
 	int res;
+	int retry;
```

Comment:
i would personally name it "reties" as this is the reties count and not binary "should I retry or not", but doesn't matter much